pkg/testutil/testutil.go

```diff
@@ -21,20 +20,12 @@ func CaptureStdout(f func()) string {
 	}()
 	os.Stdout = writer
 	log.SetOutput(writer)
-	out := make(chan string)
-	wg := new(sync.WaitGroup)
-	wg.Add(1)
-	go func() {
-		var buf bytes.Buffer
-		wg.Done()
-		_, err = io.Copy(&buf, reader)
-		if err != nil {
-			panic(err)
-		}
-		out <- buf.String()
-	}()
-	wg.Wait()
```

Comment:
I thought this was needed to prevent `writer` from hanging- as `os.Pipe` isn't buffered, we'd need to concurrently read from the reader to unblock the `writer`

Reply:
@mumoshu ok. maybe the old code is better. I will close this PR.